src/app.tsx

```diff
@@ -1,10 +1,16 @@
-import type React from 'react';
-import { st, classes } from './app.st.css';
+import { st, classes } from './style/main.st.css';
+import InstagramPost from './components/Instagran-post';
+import { MAIN_PICTURE, PROPILE_PICTURE, POST_DESCRIPTION, USER_NAME } from './constants';
 
-export interface AppProps {
-    className?: string;
-}
-
-export const App: React.FC<AppProps> = ({ className }) => {
-    return <main className={st(classes.root, className)}></main>;
+export const App = () => {
+    return (
+        <main className={st(classes.root)}>
+            <InstagramPost
+                img={MAIN_PICTURE}
+                userName={USER_NAME}
+                postDescription={POST_DESCRIPTION}
```

Comment:
If the component is instagram**Post** do you really need to specify that its a post description